source/mainnet/smart-contracts/general/contract-schema.rst

```diff
@@ -174,5 +174,4 @@ The supported sections depend on the module version. The schema version can be e
    - For 'V0' modules this is always a 'V0' schema.
    - For 'V1' modules this can be a 'V1', 'V2', or 'V3' schema.
 
-- 'V0' modules additionally support section ``concordium-schema-v1`` which always contain a 'V0' schema.
 - 'V1' modules additionally support section ``concordium-schema-v2`` which always contain a 'V1' schema.
```

Comment:
I don't think we've clearly stated that the documentation here is about V1 contracts. Perhaps it is better to simply mention that V 1-3 schemas are available on the contracts and then avoid any mention of V0/V1 contracts.